services/appveyor.js

```diff
@@ -0,0 +1,39 @@
+const loadLogos = require('../lib/load-logos');
+
+module.exports = [{
```

Comment:
Services are returned as an array to handle the cases where one service has multiple handlers (eg. NuGet has one URL for release version number, one URL for unstable version number, and URLs for download counts)

Reply:
Makes sense. How about allowing a single service to be returned as a shorthand?